app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, @content_item["title"] %>
+
```

Comment:
Oh sorry, that's my bad for putting in a screenshot of the page before we had the content item in production (it gets the breadcrumbs and e.g. the title from the content item). Because breadcrumbs are now in the content item, we can just use collections' in-built code to render them - you should see them on the [index page](https://collections-pr-3113.herokuapp.com/government/history/past-foreign-secretaries) and on the [individual pages](https://collections-pr-3113.herokuapp.com/government/history/past-foreign-secretaries/edward-wood) (links to heroku).

I've corrected the screenshot now.